### DISCUSSION TITLE
Vhf RF Gain fm

### DISCUSSION THREAD

**Discussion by iz3krk** [egzumer/uv-k5-firmware-custom#321] [General]
Hello, from the firmware v.14 the RF gain in VHF has decreased. Before I received strong signals in 70 MHZ, from the V.14 version I receive little, almost nothing. Is possible to increase RF gain in VHF band like a v.13?  Thanks

**Comment by egzumer** [comment]
The gain did not decrease. Build with AGC show data option and see for yourself.

**Reply by iz3krk** [reply]
ok, but in the next version, please try to increase a little RF Gain, if is possible

**Reply by egzumer** [reply]
In what modulation? Which gain? There are 4 different gains. There is AM fix, there is AGC. It is not that simple as you think.

**Reply by iz3krk** [reply]
74 MHZ  FM Modulation, i made a video for you and reply in other topic

**Reply by iz3krk** [reply]
This is other video of 74 mhz received with v.13 https://we.tl/t-zLoCGUAtnV
This service is impossible to receive with new version, zero signal from v.14 to v.21 (same antenna, same position, same day)

**Reply by egzumer** [reply]
With v14 the gain actually increased and not decreased [4a96dfa6ce16dfb077694a423bcadf9fcca421f2](https://github.com/egzumer/uv-k5-firmware-custom/commit/4a96dfa6ce16dfb077694a423bcadf9fcca421f2#diff-5ee282247682ee679749ecb07a880a94dfe6c86fe69c506c5e69436da0bbd059L522)

**Reply by iz3krk** [reply]
Maybe we need to decrease because maybe it's saturated

**Reply by iz3krk** [reply]
i don't know whats happend. But with the new firmware 70 mhz band are ko
